backend/post-service/src/main/java/com/speakapp/postservice/services/CommentService.java

```diff
@@ -167,4 +178,25 @@ public void deleteComment(UUID userId, UUID commentId){
 
         commentRepository.delete(commentToDelete);
     }
+
+    private List<Comment> findCommentPageByPostSorted(Post post,int firstComment, int lastComment, String sortBy, String sortOrder) {
+        String jpqlQuery = "SELECT c FROM Comment c WHERE c.post = :post ORDER BY";
```

Comment:
nie wiem dokładnie jak działa to w jpa, ale nie powinno w tym tutaj być coś w stylu `SELECT TOP {lastComment} FROM Comment c ...`?

Reply:
no to jest kwestia czy ta klasa `Query` to robi za nas. Idealnie chcielibyśmy ograniczyć to ile pobieramy rekordów z bazy danych